tests/test-files/transform/MissingImportParens.elm

```diff
@@ -0,0 +1,14 @@
+module Main exposing (x)
+
+import Import.As1 as ImportAs1
+import Import.As2 as ImportAs2 exposing A
+import ImportDotDot exposing ..
```

Comment:
I wonder if we should really allow this one (the `..` without parens) to start with.  (I'm noting it's an explicit line of code that's added to allow this.)  This seems much less likely for people to actually type it this way.  I'd rather avoid adding this on speculation, but if you think this is actually common for people that will be benefiting from this feature, then I guess it's okay.  What do you think?

Reply:
There are arguments for both sides. There is an extra branch to support it and we're not sure, how likely it is for people to use. On the other side: Having a named value working, but ".." not, is inconsistent. Why is it like this and how should the user know?

Since it would work right now, I'd go and support it.